### ISSUE TITLE
Default functions are lost

### ISSUE THREAD

**Issue by 8666** [gersilex/informational-header#1]
Can not edit the dashboard after the header is replaced with this one ..also the clock, speak, fullscreen buttons are not there anymore

**Comment by gersilex** [comment]
I had a quick discussion [in the openHAB Forums](https://community.openhab.org/t/custom-widget-informational-header/35953/9?u=gersilex) with `ysc`, the developer of HABPanel.

Because we replace the header (instead of adding functions to it), the default functionality is indeed gone. Ysc wanted to keep this in mind to make it possible for header developers to re-add the default functionality. AFAIK this was not addressed, yet. I will go through the code to see if it was already implemented and I will open a feature request at the HABPanel repo, if it hasn't.

Will report back.

**Comment by gersilex** [comment]
Opened an [issue in the HABPanel Repository](https://github.com/openhab/org.openhab.ui.habpanel/issues/318)

**Comment by gnalbandian** [comment]
Hi, are there any news regarding this?

**Comment by gersilex** [comment]
@gnalbandian Unfortunately not. Easiest way is still to change the "view" path to "edit" in the URL bar of your browser:

![image](https://user-images.githubusercontent.com/2018039/58212456-2584d100-7cef-11e9-9e62-cf17b7727a54.png)

**Comment by gnalbandian** [comment]
Yes, I do it that way...what i'm missing is the datetime  information

**Comment by gersilex** [comment]
Oh yes, totally forgot about that. But still, no solution available from me or ysc right now. :(
